css/styles-mobile-menu.css

```diff
@@ -0,0 +1,104 @@
+.mobile-menu-container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    padding: 24px 24px 40px 16px;
+    background: var(--white);
+    overflow: auto;
+    opacity: 0;
+    visibility: hidden;
```

Comment:
What is the purpose of usage both 'opacity' and 'visibility'?

Reply:
The visibility declaration used to hide the element without changing the layout of the page. Opacity used to make the element transparent for now, but the idea is that with the help of Javascript the opacity value would be changed so to make the block element sorts of appear smoothly on the screen.